packages/cloud-cognitive/src/components/ExportModal/ExportModal.js

```diff
@@ -86,13 +86,13 @@ export let ExportModal = ({
       modalHeading={modalHeading}
       onRequestSubmit={onSubmitHandler}
       onRequestClose={onCloseHandler}
-      className={`${pkg.prefix}-export-modal`}
+      className={`${pkg.prefix}--export-modal`}
```

Comment:
Some other components have been using a `blockClass` variable which would be

```js
const blockClass = `${pkg.prefix}--export-modal`;
```

then in your JSX you would just have
```JSX
<Modal
  className={cx(blockClass, `${blockClass}__another-class`)}
/>
```